app/helpers/spotlight/application_helper.rb

```diff
@@ -135,17 +135,24 @@ def available_view_fields
       current_exhibit.blacklight_configuration.default_blacklight_config.view.to_h.reject { |_k, v| v.if == false }
     end
 
-    def exhibit_stylesheet_link_tag(tag)
-      if current_exhibit_theme && current_exhibit.theme != 'default'
-        stylesheet_link_tag "#{tag}_#{current_exhibit_theme}"
+    def themed_stylesheet_link_tag(tag)
```

Comment:
The method name change here is a breaking change. Could we provide backwards compatibility and deprecation warnings so adopters know how to update?